app/uk/gov/hmrc/platformstatusfrontend/config/ErrorHandler.scala

```diff
@@ -21,11 +21,11 @@ import javax.inject.{Inject, Singleton}
 import play.api.i18n.MessagesApi
 import play.api.mvc.Request
 import play.twirl.api.Html
-import uk.gov.hmrc.play.bootstrap.http.FrontendErrorHandler
 import uk.gov.hmrc.platformstatusfrontend.views.html.error_template
```

Comment:
Can we rename the html templates to use CamelCase - which is compatible with Scala type naming?